sql/src/main/java/io/crate/analyze/repositories/RepositorySettingsModule.java

```diff
@@ -148,7 +148,6 @@ public GenericProperties dynamicProperties(GenericProperties genericProperties)
             entry("location_mode", Setting.simpleString("location_mode", "primary_only", Setting.Property.NodeScope)),
 
             // client related settings
-            entry("client", Setting.simpleString("client", "default")),
```

Comment:
we don't need to expose client setting, we use a single `default` client (same as for s3), in one of PRs we discussed that it can be a follow up task if needed to support multiple clients.